ocs_ci/deployment/deployment.py

```diff
@@ -1240,6 +1241,13 @@ def deploy_lvmo(self):
         lvmo_version_without_period = lvmo_version.replace(".", "")
         label_version = constants.LVMO_POD_LABEL
         create_catalog_source()
+        lvm_full_version = get_lvm_full_version()
+        major, minor = lvm_full_version.split("-")[1]
+        if int(minor) > 105 and major == "4.11":
+            lvmo_version_without_period = "411"
+        elif int(minor) < 105 and major == "4.11":
+            lvmo_version_without_period = "411-old"
```

Comment:
We need it for a workaround to keep developing automation. Above 4.11-105 the label change is there but the feature doesn't work. Below 4.11-105 the feature is working but the labels are old. This enable to choose the right key for the right labels so we can deploy for now below 4.11.-105 to continue some automation effort.

Reply:
Test with the latest when it can be tested and not broken. I want also the option to deploy older builds - now because they are the only one that works and maybe in the future for comparison.

Reply:
please open a workaround issue that will track the removal of these conditions

Reply:
update #6074